mock/mock.spec

```diff
@@ -257,6 +259,7 @@ pylint-3 py/mockbuild/ py/*.py py/mockbuild/plugins/* || :
 %config(noreplace) %{_sysconfdir}/%{name}/*.ini
 %config(noreplace) %{_sysconfdir}/pam.d/%{name}
 %config(noreplace) %{_sysconfdir}/security/console.apps/%{name}
+%{_sysusersdir}/mock.conf
```

Comment:
Well, I know @xsuchy requested this, but ... do we need this?

Reply:
Ah, not needed by the scriptlet itself - but it is used by systemd (or the deamon) which may take care of useradd/groupadd.